images/kubeflow-pipelines/configs/latest.frontend.apko.yaml

```diff
@@ -0,0 +1,20 @@
+contents:
```

Comment:
This should use the config/main.tf convention, instead of having the config only specified in YAML.

Reply:
done.